pgwidget/pgwidget_core.py

```diff
@@ -304,7 +308,10 @@ def _round_cursor_position_to_nearest_letter(self,pos):
                 letter_row -= 1
         
         # total_shown_text_length=self.myfont.size(self.shown_text)[0]
-        total_shown_text_length = self.myfont.size(shown_text_rows[letter_row])[0]
```

Comment:
What could cause this to end with error? `letter_row` being higher than number of rows in `shown_text_rows`?